session/tidb.go

```diff
@@ -42,6 +42,12 @@ import (
 	"go.uber.org/zap"
 )
 
+func init() {
+	domain.DomainMapDelete = func(store kv.Storage) {
+		domap.Delete(store)
```

Comment:
There is another global map `storeBootstrapped` which hold a `storeUUID -> bool`. `ResetStoreForWithTiKVTest` is a similar  function to clean the two maps.